pkg/ecdsa/tss/marshaling_test.go

```diff
@@ -176,3 +176,37 @@ func TestFuzzAnnounceMessageRoundtrip(t *testing.T) {
 func TestFuzzAnnounceMessageUnmarshaler(t *testing.T) {
 	pbutils.FuzzUnmarshaler(&AnnounceMessage{})
 }
+
+func TestLiquidationRecoveryMessageRoundtrip(t *testing.T) {
```

Comment:
Maybe `TestFuzzLiquidationRecoveryMessageRoundtrip` to keep it aligned with the rest of fuzz tests?